docs/ocean/design_docs/template.rst

```diff
@@ -0,0 +1,96 @@
+
+Some descriptive title
+======================
+
+date: YYYY/MM/DD
+
+Contributors:
+
+
+
+Summary
+-------
+
+The purpose of this section is to summarize what capability is to be added to
+MPAS-Model through this design process. It should be clear what new code will do
+that the current code does not. Summarizing the primary challenges with respect
+to software design and implementation is also appropriate for this section.
+Finally, this statement should contain a general statement with regard to what
+is "success."
+
+
+Requirements
+------------
+
+Requirement: name-of-requirement-here
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Date last modified: YYYY/MM/DD
+
+Contributors: (add your name to this list if it does not appear)
+
+
+Each requirement is to be listed under a "section" heading, as there will be a
+one-to-one correspondence between requirements, design, proposed implementation
+and testing. Requirements should not discuss technical software issues, but
+rather focus on model capability. To the extent possible, requirements should
+be relatively independent of each other, thus allowing a clean design solution,
+implementation and testing plan.
+
+
+Algorithmic Formulations (optional)
+-----------------------------------
+
+Design solution: short-description-of-proposed-solution-here
```

Comment:
@xylar Should the `Design solution` section be moved to `Design and Implementation`? Couldn't quite figure out how to formally "suggest" this.

Reply:
I agree, this should change.  Whoever originally designed this template used this naming convention and I agree it's confusing.  I think `Implementation` is the correct name in the next section. So the sections should instead presumably be `Algorithm Design` (which could be the section and the subsection both) and `Implementation` (again, section and subsection).  Does that make sense to you?

Reply:
Yes, that sounds appropriate to me.

Reply:
By the way, you're right.  "suggest" only helps when you're making minor tweaks to the text, not moving sections and such.